### PR TITLE
New phishing layout

### PR DESCRIPTION
**Pull request by idovandijk**
## Status
- [x] In Progress

## Related Issues
fixes: https://github.com/demisto/etc/issues/46959 (tentative)
fixes: https://github.com/demisto/etc/issues/45588
fixes: https://github.com/demisto/etc/issues/43944
fixes: https://github.com/demisto/etc/issues/45030

## Description
- Created a new layout - relevant to `Phishing - Generic v3` and `Phishing - Core v2`. The layout includes many changes, some of which are:
  - An interactive section in the layout for deleting the email that was reported.
  - A new section for machine-learning checks
  - A representation of the domain-squatting check
  - More important things will stand out first, added a separate tab for advanced fields
  - Removed irrelevant things like the non-working widget, critical assets grid and metadata of attachments of the EML/MSG file which are already shown in the Incident Files section
- Fixed an issue where the "Email Text" section would sometimes be empty.

## Minimum version of Cortex XSOAR
6.1.0

## Does it break backward compatibility?
No

### CODE REVIEW

File: Packs/CommonTypes/pack_metadata.json [hunk 1]
```diff
@@ -2,9 +2,9 @@
     "name": "Common Types",
     "description": "This Content Pack will get you up and running in no-time and provide you with the most commonly used incident & indicator fields and types.",
     "support": "xsoar",
-    "currentVersion": "3.2.26",
+    "currentVersion": "3.2.27",
     "author": "Cortex XSOAR",
-    "url": "https://www.paloaltonetworks.com/cortex",
+    "url": "https:\/\/www.paloaltonetworks.com\/cortex",
```
Comment (idovandijk) [inline Packs/CommonTypes/pack_metadata.json:7]: @jochman I'm using your version of the SDK with the new JSON loader. Could this be related?

Comment (jochman) [reply Packs/CommonTypes/pack_metadata.json:7]: that's what happened when you ran format?

Comment (jochman) [reply Packs/CommonTypes/pack_metadata.json:7]: Also, what is the impact when you're uploading it to XSOAR? If there is any.

Comment (jochman) [reply Packs/CommonTypes/pack_metadata.json:7]: image
As you can see, the python's built-in is loading it as intended to be.

The only side-effect I can see is if XSOAR isn't parsing the json (go parser) as intended.

Comment (idovandijk) [reply Packs/CommonTypes/pack_metadata.json:7]: I assume that the version bump did that and not the format, don't know what the effect would be on XSOAR

Comment (idovandijk) [reply Packs/CommonTypes/pack_metadata.json:7]: @dorschw have u seen this before? happened after bumping version

**Comment by idovandijk** [comment]
@jrosenpanw can you please assign someone for reviewing RN? Thanks

File: Packs/CommonTypes/pack_metadata.json (hunk 1 above)
Comment (jochman) [reply Packs/CommonTypes/pack_metadata.json]: It should be okay with the escape character.

Comment (dorschw) [reply Packs/CommonTypes/pack_metadata.json]: solved in https://github.com/demisto/demisto-sdk/pull/2005 FYI @idovandijk

**Review by tomer-pan — APPROVED** [review]

**Pull request merged**